taichi/backends/vulkan/spirv_header.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#ifdef TI_WITH_VULKAN
+
+#include <spirv/1.0/spirv.hpp>
```

Comment:
Is this a header from the SDK?

Reply:
It's from https://github.com/KhronosGroup/SPIRV-Headers. According to https://github.com/KhronosGroup/glslang

> If people are only using this location to get spirv.hpp, I recommend they get that from SPIRV-Headers instead.

Reply:
I think the include path on Windows' SDK is `SPIRV-Headers/spirv.hpp`, same would be true if it's a git submodule

Reply:
I don't think `spirv.hpp` is exposed as a  top-level header. Also if you install it to the default header search dir on Linux, it will be `/usr/local/include/spirv/1.0/spirv.hpp`.